collection_json.py

```diff
@@ -140,6 +140,29 @@ class Template(ComparableObject):
 
     """Object representing a Collection+JSON template object."""
 
+    @staticmethod
+    def from_json(data):
+        """Return a template instance.
+
+        Convenience method for parsing 'write' responses which should only contain the template,
+        not in a collection object.
```

Comment:
I think this line has a typo, should it read 'not a collection object.' instead?